### PR TITLE
add doc for enable observability from ui

### PR DESCRIPTION
**Pull request by songleo**
https://github.com/open-cluster-management/backlog/issues/13369

Signed-off-by: Song Song Li <ssli@redhat.com>

### CODE REVIEW

File: observability/how-to-enable-observability-service-using-the-web-console.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        <img src="./images/secret-page-for-img-pull.png" alt="image pull secret page" width="1600" height="390" />
+    </p>
+
+   - Enter your image pull secret details. Click `Create`:
+
+    <p align="center">
+        <img src="./images/create-pull-secret.png" alt="Create image pull secret" width="900" height="720" />
+    </p>
+
+3. Create your object storage secret `thanos-object-storage` in `open-cluster-management-observability` project, Observability service supports the following stable object stores:
```
Comment (clyang82) [inline observability/how-to-enable-observability-service-using-the-web-console.md]: It is better to link the previous section about  object storage. we do not need to declare here again.

File: observability/how-to-enable-observability-service-using-the-web-console.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+   - Click `Create Project` and enter your project name `open-cluster-management-observability`. Click `Create`:
+
+    <p align="center">
+        <img src="./images/create-project.png" alt="Create project" width="600" height="400" />
+    </p>
+
+    You must use `open-cluster-management-observability` as the project name. Observability service does not support installation to other projects.
+
+2. Create your image pull secret `multiclusterhub-operator-pull-secret` in `open-cluster-management-observability` project:
```
Comment (clyang82) [inline observability/how-to-enable-observability-service-using-the-web-console.md:28]: Here you may mention that get the pull secret content from `secret/pull-secret -n openshift-config`

**Comment by dockerymick** [comment]
Shared with @songleo that we need to create a new PR because we (doc team), use a different file extension. We use ascii (.adoc). I would like to keep this PR open since @clyang82 is doing a review, I will create a new PR based on the comments and changes here.

File: observability/how-to-enable-observability-service-using-the-web-console.md (hunk 1 above)
Comment (songleo) [reply observability/how-to-enable-observability-service-using-the-web-console.md]: OK, let me update it. thanks.

**Comment by songleo** [comment]
@dockerymick I have updated the doc. I think you can new a new PR for it. thanks.

**Comment by dockerymick** [comment]
@songleo @clyang82 Thank you both for your patience, review, and suggestions. I have created a new PR [here](https://github.com/open-cluster-management/rhacm-docs/pull/2332) and will close this PR

**Pull request closed without merging**